src/Derivative.h

```diff
@@ -35,10 +35,6 @@ class Derivative {
     Func operator()(const Buffer<> &buffer) const;
     Func operator()(const Param<> &param) const;
 
-    /** Get the unbounded Func, if any. If no such Func (ie no boundary condition
-     * was applied), equivalent to just calling operator(). */
-    Func get_unbounded(const Func &func, int update_id = -1) const;
-
```

Comment:
The funcs method below can go away now. The fit_function example will need fixing

Reply:
Fixed.